### PR TITLE
LVUIDF-49: created project structure

### PR DESCRIPTION
**Pull request by yurko**
https://ssu-jira.softserveinc.com/browse/LVUIDF-49

Created project structure
Removed unneeded files
Added .gitignore file
Cleaned unnecessary comments
Removed turbolinks
Edited Gemfile to include only needed gems

To review:
- [x] @rtriska
- [x] @vyshnevska

Optional:
- [ ]  @irynar 

To merge:
- [x] @izagor

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ApplicationHelper
```
Comment (rtriska) [inline app/helpers/application_helper.rb]: If module is empty we don't need it, till we have no methods to put inside. Remove it

**Comment by izagor** [comment]
@yurko please create 1 commit - squash the changes.

**Comment by vyshnevska** [comment]
lgtm

**Comment by rtriska** [comment]
looks awesome :man:

**Comment by izagor** [comment]
Adding the changes into the development branch.

**Pull request merged**